lib/modules/dosomething/dosomething_api/resources/campaign_resource.inc

```diff
@@ -58,7 +58,7 @@ function _campaign_resource_definition() {
             'name' => 'count',
             'description' => 'The number of campaigns to retrieve.',
             'optional' => TRUE,
-            'type' => 'string',
+            'type' => 'int',
```

Comment:
Nothing really broke. I believe in the each resources respective class Transformer we cast the value to an integer anyway, to make sure it's an int; but figured I'd update this to be consistent and so that Services will know ahead of time if an `int` is not passed.